code/style.css

```diff
@@ -9,50 +11,141 @@ p {
   background-color: #fff;
 }
 
+.header-styling {
```

Comment:
Not a requirement, but the positioning of the header was a bit off in my browser. Maybe center the header so that it shows correctly when making wider/narrower